Content.Server/Fluids/EntitySystems/DrainSystem.cs

```diff
@@ -12,6 +26,88 @@ public sealed class DrainSystem : EntitySystem
         [Dependency] private readonly EntityLookupSystem _lookup = default!;
         [Dependency] private readonly SolutionContainerSystem _solutionSystem = default!;
         [Dependency] private readonly SharedAmbientSoundSystem _ambientSoundSystem = default!;
+        [Dependency] private readonly AudioSystem _audioSystem = default!;
```

Comment:
SharedAudioSystem (its better to use the shared version in dependencies since it makes refactoring easier)